docs/specs/xref.yml

```diff
@@ -1982,3 +1981,40 @@ outputs:
   .errors.log: |
     {"message_severity":"warning","code":"duplicate-uid","message":"UID 'a' is duplicated in 'test.yml(2,6)', 'test.yml(4,8)'.","file":"test.yml","line":4,"column":8}
     {"message_severity":"warning","code":"duplicate-uid","message":"UID 'a' is duplicated in 'test.yml(2,6)', 'test.yml(4,8)'.","file":"test.yml","line":2,"column":6}
+---
+# Xref property field references to uid
```

Comment:
can you mention that we resolve as absolute link specifically for TSType and Architecture?